### ISSUE TITLE
[BUG] "Unicode" output encoding mixes newlines

### ISSUE THREAD

**Issue by aaaxx** [CCExtractor/ccextractor#767]
CCExtractor version: **0.85**
Platform: Windows

Using the combination of Unicode encoding and "DOS/Windows (CRLF)" line terminators creates files that use both CR and CRLF newlines. (The "UNIX (LF)" option works fine, though.)

![screenshot of an affected file opened in a text editor, with whitespace characters visible](https://user-images.githubusercontent.com/6186480/29266377-d5d12a94-80e4-11e7-9ca7-99ae5f9c21f4.png)

**Comment by saurabhshri** [comment]
@aaaxx What exact command did you use? Also, did the video by any change had bitmap subtitles (e.g. was it a DVB sample?).

I have made a PR which should hopefully fix this (as the context based newlines were missing).

**Comment by aaaxx** [comment]
```
ccextractorwin.exe  --gui_mode_reports  -autoprogram  -out=srt -bom -unicode [+input files]
```

The source is a retail DVD (multiple VOB files).

Note that it happens only with `-unicode`, not with `-utf8` and `-latin1`.

**Comment by aaaxx** [comment]
And yes, it also had bitmap subtitles.

**Comment by saurabhshri** [comment]
@aaaxx Please try again with the latest code from the repo, and if you still see the mixed line endings, feel free to re-open the issue.

**Comment by aaaxx** [comment]
Sorry, I'm not geeky enough to know how to compile things, but if you can be bothered to make an exe, I'll test it.

**Comment by Pirater12** [comment]
![task_2_3](https://user-images.githubusercontent.com/16360444/33383558-aa329c88-d549-11e7-92fc-982dbbb8447e.png)
This issue was fixed with pr #783.

**Comment by saurabhshri** [comment]
@Pirater12 Thank you for validating the fix! :)
